### PR TITLE
BAH-378 Setup bed management OWA

### PR DESCRIPTION
**Pull request by sanishmaharjan**
[BAH-378](https://bahmni.atlassian.net/browse/BAH-378) Design and integrate bed management admin UI
@mks-d Please code review.

### CODE REVIEW

**Comment by sanishmaharjan** [comment]
@mks-d Updated README.md file with Development Instructions.

File: owa/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+```
+
+3. Build & compile code
+```bash
+# npm command
+$ npm run webpack
+$ npm run build-webpack # pack with Compress for production
+
+# yarn command
+$ yarn webpack
```
Comment (mks-d) [inline owa/README.md]: @sanishmaharjan and then, where do we see the app for debugging?

Comment (sanishmaharjan) [reply owa/README.md]: I use Chrome React Developer Tool for debugging.

Comment (mks-d) [reply owa/README.md]: I mean, what's the URL to go to?

Comment (sanishmaharjan) [reply owa/README.md]: - https://192.168.33.10/owa/bedmanagement/admissionLocations.html
Since I have set 
App Base URL: https://192.168.33.10/owa/

Comment (mks-d) [reply owa/README.md]: No I mean the standalone URL to view the React app. Or then why would anyone run `yarn webpack`, what does this actually do?

Comment (sanishmaharjan) [reply owa/README.md]: "yarn webpack" command bundle reactJs code and watch it change so that code changes will reflect.

**Comment by dkayiwa** [comment]
@sanishmaharjan it would be great if you also included the ticket url as advised at https://wiki.openmrs.org/display/docs/Pull+Request+Tips
It helps the reviewer quickly jump to the ticket that is being worked on. 😊

**Pull request merged**